diffkemp/simpll/SourceCodeUtils.cpp

```diff
@@ -50,6 +50,8 @@ std::unordered_map<std::string, MacroElement> getAllMacrosOnLine(
             // from the left, record all such substrings and test them using the
             // macro map provided in the function arguments.
             std::string macroBody = Entry.second.body.str();
+            expandCompositeMacroNames(pairZip(Entry.second.params,
+                                Entry.second.args), macroBody);
```

Comment:
This could be aligned in a nicer way:

    expandCompositeMacroNames(pairZip(Entry.second.params,
                                      Entry.second.args), 
                              macroBody);